config/locales/en.yml

```diff
@@ -216,14 +216,26 @@ en:
       private_access_link_help: "Use this link to continue and manage publisher registration"
       salutation: "Hi %{name},"
       signature: "The Brave Payments Team"
+      post_script: |
+        <p>Got questions? Visit <a href="https://basicattentiontoken.org/">basicattentiontoken.org</a> to learn more about how the micropayments work.
+        <p>We look forward to having you join the thousands of domains already receiving funds from their audience.</p>
       button_not_working: "If the button above doesn't work, paste this link into your web browser: "
       erroneous_email_help: "If you did not make this request, you can ignore this email or contact us at "
       terms_of_service: "Terms of Service"
       medium: "Medium"
       twitter: "Twitter"
       reddit: "Reddit"
       rocketchat: "Rocket.Chat"
-
+    
+    unverified_domain_reached_threshold:
+      subject: "Money is waiting for [%{publication_title}], verify your site with Brave to claim your funds!"
```

Comment:
maybe we don't need the `[]`.. they look a little funny like it's a template email